var/spack/repos/builtin/packages/embree/package.py

```diff
@@ -37,23 +45,33 @@ def cmake_args(self):
             "-DEMBREE_TUTORIALS=OFF",
             "-DEMBREE_IGNORE_CMAKE_CXX_FLAGS=ON",
             self.define_from_variant("EMBREE_ISPC_SUPPORT", "ispc"),
+            self.define("EMBREE_TBB_ROOT", spec["tbb"].prefix),
```

Comment:
Is this option supported by all versions?

Reply:
As far as I can see, this setting was introduced in https://github.com/embree/embree/commit/fc9c94993aa32727a23562a078867cbbed1fdbb3 and released with embree 2.11.0, so I would say yes: all checksummed versions (3.7 and newer) should support it, even though only 3.8.0 starts mentioning it in README.md.